image/project/Dockerfile

```diff
@@ -16,12 +16,16 @@ RUN npm install --production --no-package-lock && \
     [ ! -d /project/function/node_modules ] && mkdir /project/function/node_modules
 
 # Copy the dependencies into an alpine image
-FROM node:12-alpine
-COPY --chown=node:node . /project
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
```

Comment:
do we want to target `8.1` or `rhel8` in general? if the latter we might want to change the tag to `8`. I hope there's one like that :)

Reply:
I've been using `8`. It selects the latest 8.x at the time of build.

Reply:
That's a good point. The `8.1` tag is floating, but when `8.2` is released, we won't pick it up. There is no simple `8` tag, but there is `latest` which is essentially what you are asking for (the 8 is in the repository name, `ubi8`). If we can safely assume that minor releases do not include breaking changes, then I'd say this is OK.

Reply:
Yes. Sorry.. when I replied above, I hadn't checked what our productization Dockerfile's actually use. :/

Currently they use:

```
FROM ubi8-minimal:8-released
```

However, for some reason, that tag is not available on registry.access.redhat.com. It's worth noting that registry.access.redhat.com will eventually be retired.. see https://www.redhat.com/en/blog/transitioning-red-hat-container-registry.

As a an aside, I'm switching the productization dockerfiles to use an explicit host/path to refer to images. What they use now makes it hard to build the images locally.

Reply:
If registry.access.redhat.com is going to be retired what should I use ? registry.redhat.io? Don't I need credentials for that?

Reply:
Should I use `registry.access.redhat.com/ubi8/ubi-minimal:latest`, then?

Reply:
If you need to avoid using credentials and want to use a public registry, I suggest we use registry.access.redhat.com for now. Of course, with the understanding that it may need to change in the future.

Reply:
@lance I changed tag to latest is it good now?

Reply:
@warrenvw btw when building I got some warnings:
```
[Docker] (process:1): librhsm-WARNING **: 10:21:28.796: Found 0 entitlement certificates                                             
[Docker]                                                                                                                             
[Docker] (process:1): librhsm-WARNING **: 10:21:28.801: Found 0 entitlement certificates                                             
[Docker]                                                                                                                             
[Docker] (process:1): libdnf-WARNING **: 10:21:28.802: Loading "/etc/dnf/dnf.conf": IniParser: Can't open file
```
Is it anything serious?

Reply:
@matejvasek let's hear from @warrenvw before merging, but otherwise looks great!

Reply:
These are warnings, and should not present any problem. See https://access.redhat.com/solutions/4643601 for more info.

You may want to consider replacing `ubi8/ubi-minimal` with `ubi8/ubi` as it compiles without warnings.

FWIW, I have confirmed that by default, `/etc/dnf/dnf.conf` exists in the `ubi8/ubi` container, but not in `ubi8/ubi-minimal`.

ubi8/ubi-minimal:

```
[root@18462e6b57fe /]# cd /etc/dnf
[root@18462e6b57fe dnf]# ls
modules.d  protected.d
```

ubi8/ubi:

```
[root@d9dd357fb095 dnf]# pwd
/etc/def
[root@d9dd357fb095 dnf]# ls
aliases.d  dnf.conf  modules.d	modules.defaults.d  plugins  protected.d  vars
[root@d9dd357fb095 dnf]# cat dnf.conf 
[main]
gpgcheck=1
installonly_limit=3
clean_requirements_on_remove=True
best=True
skip_if_unavailable=False
```